nengo_viz/static/viz.js

```diff
@@ -454,3 +454,31 @@ VIZ.make_colors = function(N) {
     }
     return c;
 }
+
+//Check if value is a number
+VIZ.is_num = function(value){
+    if (!(isNaN(value)) && !(value.trim() == '') ) {
+        return true;
+    }
+    else{
+        alert('Not a valid number! Please try again.');
```

Comment:
Yikes!  I would really not expect a helper function like ```is_num``` to pop up an alert on a failure.  That should be the responsibility of whoever calls ```is_num```.